Documentation/building/unix-instructions.md

```diff
@@ -102,6 +102,9 @@ ln -s /usr/local/opt/openssl/lib/pkgconfig/libssl.pc /usr/local/lib/pkgconfig/
 ln -s /usr/local/opt/openssl/lib/pkgconfig/openssl.pc /usr/local/lib/pkgconfig/
 ```
 
+Alternatively, instead of butchering /usr/local/ like a savage. Invoke cmake with `OPENSSL_ROOT_DIR=/usr/local/Cellar/openssl/*/`.
```

Comment:
What does the `*` do when there's more than one?

Reply:
> Alternatively, instead of butchering /usr/local/ like a savage.

Please change this to something like "Alternatively, to avoid impacting /usr/local/, invoke cmake with `OPENSSL_ROOT_DIR=/usr/local/Cellar/openssl/*/`."

Reply:
@bartonjs The * is to deal with users having different version of openssl installed.

This is to deal with the fact that we don't dictate a specific version of openssl to be installed.

Hmmm, funny thing. This approach is wrong. It works my laptop but now checking on my desktop, it would fail as, for some reason, brew have multiple versions.

I'll have to fix the wording to deal with that.